packages/cosmwasm/src/signingcosmwasmclient.ts

```diff
@@ -175,22 +176,22 @@ export class SigningCosmWasmClient extends CosmWasmClient {
    *
    * @param apiUrl The URL of a Cosmos SDK light client daemon API (sometimes called REST server or REST API)
    * @param senderAddress The address that will sign and send transactions using this instance
-   * @param signCallback An asynchonous callback to create a signature for a given transaction. This can be implemented using secure key stores that require user interaction.
+   * @param signer A wallet provider which can provide signatures for transactions, potentially requiring user input.
```

Comment:
"A wallet provider" -> "An implementation of OfflineSigner" (or something like that)